sale_order_line_cancel/wizards/sale_order_line_cancel.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Okia SPRL
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class SaleOrderLineCancel(models.TransientModel):
+    _name = "sale.order.line.cancel"
+    _description = "Cancel Remaining Wizard"
+
+    def _get_sale_order_line(self):
+        active_id = self._context.get("active_id")
```

Comment:
`_context` is deprecated

```suggestion
        active_id = self.env.context.get("active_id")
```